tests/quantization/compressed_tensor/test_compressed_tensors.py

```diff
@@ -0,0 +1,53 @@
+import gc
+import unittest
+
+from transformers import AutoModelForCausalLM, AutoTokenizer
+from transformers.testing_utils import require_compressed_tensors, require_torch
+from transformers.utils import is_torch_available
+
+
+if is_torch_available():
+    import torch
+
+
+@require_compressed_tensors
+@require_torch
+class CompressedTensorsTest(unittest.TestCase):
+    quantized_model_name = "nm-testing/tinyllama-oneshot-w8a8-test-static-shape-change-v3"
```

Comment:
Let's add more tests to fully cover all compression config ! This will help a lot to detect breaking changes in the future.

Reply:
@bfineran we could add some from the vllm [weight loading test](https://github.com/vllm-project/vllm/blob/3ac50b47d0f718364d05b1bcb93743e15be6a37c/tests/weight_loading/models.txt#L7-L15):
```
compressed-tensors, nm-testing/tinyllama-oneshot-w8w8-test-static-shape-change, main
compressed-tensors, nm-testing/tinyllama-oneshot-w8-channel-a8-tensor, main
compressed-tensors, nm-testing/tinyllama-oneshot-w8a8-dynamic-token-v2, main
compressed-tensors, nm-testing/tinyllama-oneshot-w8a8-channel-dynamic-token-v2, main
compressed-tensors, nm-testing/tinyllama-oneshot-w4a16-group128-v2, main
compressed-tensors, nm-testing/tinyllama-oneshot-w8a16-per-channel, main
compressed-tensors, nm-testing/Meta-Llama-3-8B-FP8-compressed-tensors-test, main
compressed-tensors, nm-testing/Phi-3-mini-128k-instruct-FP8, main
compressed-tensors, neuralmagic/Phi-3-medium-128k-instruct-quantized.w4a16, main
```

Llama and Phi-3 should cover dealing with unmerged and merged qkv layers